pkgs/tools/system/auto-cpufreq/default.nix

```diff
@@ -1,17 +1,21 @@
-{ lib, python3Packages, fetchFromGitHub, substituteAll }:
+{ lib, pkgs, python310Packages, fetchFromGitHub, substituteAll }:
 
-python3Packages.buildPythonPackage rec {
+python310Packages.buildPythonPackage rec {
```

Comment:
Why Python 3.10 and not the default?

Reply:
No reason in particular. Currently, they are on the same version but I assume the default `python3` will be updated eventually?

Reply:
Yes, the default will be updated eventually. I believe we try to keep most Python applications on the default version to reduce duplicated dependency packages.

Reply:
gotcha. I'll revert this